ui/panels-plugin/src/plugins/time-series-table/DataTable.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright 2024 The Perses Authors
```

Comment:
(Not sure where it would go in this file :D) Any chance to have some visual difference (e.g color and/or boldness) between label values and label names, like prometheus is doing? 
![image](https://github.com/perses/perses/assets/7058693/894be691-1339-4122-98b2-790e645cc25c)

At least for me it's a good helper for readability 😄

Reply:
Okay I see now, I took as input the formattedName which is just a string. I think I have the labels somewhere. Just need to do the same logic as them rebuilding the from labels

Thanks to that I discovered a bug in my implementation haha 
![image](https://github.com/perses/perses/assets/10258608/4c913ca5-9c46-4d5d-9b9a-c88ba4f4f352)

Reply:
argh no ok this is something new legend use the same than me..
That's interesting because I wouldn't expect to have this result in this case : 
![image](https://github.com/perses/perses/assets/10258608/5a266551-d069-45b0-82d5-7eb763ccc557)
I'd expect {} and not the repeat of the query. Strange

Reply:
As discussed in private, I will let it like that for now. And make a separate PR to handle that directly from labels. 

It can be fancy as well to extend this formatting to legends and tooltips. IMHO I don't really understand why this part is in the prometheus plugin. Is it really proper to prometheus to make such a display of labels? I would personally move it up to a shareable component. Free to use by any timeseries panel and taking labels + optional formatter to render a pretty line. So that Legend, Tooltip, and TimeSeriesTable could reuse it and have the same result.

Reply:
Regarding the default value to being the query. This is an assume choice made before us. https://github.com/perses/perses/pull/1045

I'd disagree on that but let's discuss in the future PR that will possibly change this behaviour.